app/models/minion.rb

```diff
@@ -0,0 +1,4 @@
+# Minion represents the minions that have been registered in this application.
+class Minion < ApplicationRecord
+  default_scope { order hostname: :asc }
```

Comment:
I'm a little late but I suggest we avoid default_scope in general because it is hard to override. We can always use a named scope instead.